package_manifest.yaml

```diff
@@ -200,13 +204,17 @@ foreman_nonscl_packages:
       - koji-foreman
     nightly_releaser: koji-foreman-jenkins
     repoclosure_config: repoclosure/yum_el7.conf
+    repoclosure_target_repos:
+      - downloaded_rpms
+      - el7-foreman-nightly
+      - el7-foreman-plugins-nightly
```

Comment:
Does this mean we no longer check repoclosure for nonscl without plugins present? Should we have a separate run for that so you never break repoclosure on nightly, but can break plugins and katello. For example when you're doing the fog migration.

Reply:
This is a good point. My intention here was "did a change in nonscl break plugins?", but this obviously also makes plugins resolve the deps...

Why are we always finding more layers of complexity in this?

Anyways, this PR will be closed until we sort out https://github.com/theforeman/obal/pull/186